src/platform/plugins/shared/home/public/application/components/tutorial/replace_template_strings.ts

```diff
@@ -10,21 +10,50 @@
 import { Writer } from 'mustache';
 import { getServices } from '../../kibana_services';
 
-const TEMPLATE_TAGS = ['{', '}'];
+const TEMPLATE_TAGS: [string, string] = ['{', '}'];
+interface Context {
+  lookup(name: string | number): string;
+  parent: Context | undefined;
+  cache: Record<string, any>;
+  view: {
```

Comment:
doesn't look like we need all of this? let's just type lookup inside escapedValue inline. and keep only the config part and tie it to the `variables` below